docs/reference/airnode/latest/understand/configuring.md

```diff
@@ -331,6 +331,58 @@ Remember that an Airnode's config.json file can have more than one OIS object
 and that these endpoints can be triggers for `rrp`, `http`, and/or
 `httpSignedData` as desired.
 
+#### Considerations: cached responses
+
+When setting `triggers.rrp[n].cacheResponses = true` Airnode has the ability to
+cache API responses, per request ID, to the local filesystem. This only applies
```

Comment:
There isn't any equal signs used in the config, also remove "has the ability" given you're setting to true, so I suggest:
```suggestion
When setting `triggers.rrp[n].cacheResponses` to `true` Airnode caches API responses
on the local filesystem by request ID. This only applies
```